app/src/main/java/com/pr0gramm/app/ui/fragments/FeedFragment.java

```diff
@@ -10,25 +10,14 @@
 import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.SearchView;
 import android.util.Log;
-import android.view.Display;
-import android.view.LayoutInflater;
-import android.view.Menu;
-import android.view.MenuInflater;
-import android.view.MenuItem;
-import android.view.View;
```

Comment:
Sorry kann hier auf mobile irgendwie nicht überall Kommentare hinzufügen. 

Kannst du die refresh Funktion aus dem swipeView heraus ebenfalls verwenden, oder mithilfe der refresh Funktion den refresh-inicator des swipeViews aktivieren? 

Außerdem wäre es super, wenn zu den Button in den Einstellungen aktivierbar machen könntest.